ci-operator/step-registry/ipi/deprovision/azure/resourcegroup/ipi-deprovision-azure-resourcegroup-commands.sh

```diff
@@ -12,17 +12,22 @@ CONFIG="${SHARED_DIR}/install-config.yaml"
 existing_rg=$(/tmp/yq r "${CONFIG}" 'platform.azure.resourceGroupName')
 
 # az should already be there
-az version
+command -v az
 
 # set the parameters we'll need as env vars
 AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
-AZURE_AUTH_CLIENT_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientId)"
-AZURE_AUTH_CLIENT_SECRET="$(cat ${AZURE_AUTH_LOCATION} | jq -r .clientSecret)"
-AZURE_AUTH_TENANT_ID="$(cat ${AZURE_AUTH_LOCATION} | jq -r .tenantId)"
+AZURE_AUTH_CLIENT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientId)"
+AZURE_AUTH_CLIENT_SECRET="$(<"${AZURE_AUTH_LOCATION}" jq -r .clientSecret)"
+AZURE_AUTH_TENANT_ID="$(<"${AZURE_AUTH_LOCATION}" jq -r .tenantId)"
 
 # log in with az
 az login --service-principal -u "${AZURE_AUTH_CLIENT_ID}" -p "${AZURE_AUTH_CLIENT_SECRET}" --tenant "${AZURE_AUTH_TENANT_ID}"
 
 # delete resource group in case it still exists
 echo "Deleting resource group: ${existing_rg}"
-az group delete -y --no-wait -n "${existing_rg}" &>/dev/null
+if [ "$(az group exists -n "${existing_rg}")" == "true" ]
+
+then
+	az group delete -y -n "${existing_rg}"
+
+fi
```

Comment:
Drop the extra vertical white space.
```suggestion
if [ "$(az group exists -n "${existing_rg}")" == "true" ]
then
	az group delete -y -n "${existing_rg}"
fi
```